enterprise/server/registry/registry_server.go

```diff
@@ -3,9 +3,11 @@ package registry
 import (
 	"flag"
 	"fmt"
+	"io/ioutil"
```

Comment:
ioutil is dead, long live io

https://pkg.go.dev/io#ReadAll

Reply:
Done